packages/styled-ui-docs/pages/icon.mdx

```diff
@@ -12,7 +12,7 @@ import { Icon } from '@trendmicro/react-styled-ui';
 
 ## Usage
 
-Use an icon by passing the `name` prop. This name must match an icon key defined in `theme.icons`. By default, the icon inherits the font size and color of its parent.
+Use an icon by passing the `icon` prop. This `icon` property value must match an icon key defined in `theme.icons`. By default, the icon inherits the font size and color of its parent.
```

Comment:
I think we should use size instead of font size here since this Icon is made of svg instead of real fonts.